firebase-admin/runtime/src/main/java/io/quarkiverse/googlecloudservices/firebase/admin/runtime/FirebaseAuthConfig.java

```diff
@@ -47,6 +47,61 @@ public interface AuthConfig {
          */
         Optional<String> rolesClaim();
 
+        /**
+         * Returns the session cookie configuration
+         */
+        SessionCookie sessionCookie();
+
+        public interface SessionCookie {
+
+            /**
+             * Name to use for session cookies (see <a href="https://firebase.google.com/docs/auth/admin/manage-cookies#java_2">
+             * Manage session cookies</a>)
+             */
+            Optional<String> name();
+
+            /**
+             * The expiration duration of the session cookie. Uses {@link java.time.Duration#parse(CharSequence)} to
+             * get a duration for the expiration. See the JavaDoc of this method for the format of this value.
+             * <p>
+             * Defaults to 5 days.
+             */
+            @WithDefault("P5")
+            Optional<String> expirationDuration();
+
+            /**
+             * Perform an additional check to see if the session was revoked
+             */
+            @WithDefault("true")
+            Boolean checkRevoked();
+
+            /**
+             * Validate the expiration date of the token.
+             */
+            @WithDefault("false")
```

Comment:
Is it aligned with what other security extensions are doing?

Reply:
btw @sberyozkin could you also let me know what you prefer here. Basically what this flag does:

- if false, the user will get a session token regardless of the remaining validity of the original JWT token
- if true, the user will only get a session token if his JWT token has a minimum validity

The default is currently false, as per the referenced google docs. Changing it is easy, but that would require we always specify a minimum validity (which I don't think we can set a default for). 

Agreed to leave it off by default and allow people with higher security requirements to specify this themselves?